docs/contributing/sla.md

```diff
@@ -0,0 +1,67 @@
+---
+id: sla
+title: Contribution SLA
+---
+
+This is a Service Level Agreement (SLA) between the contributor and the Cortex XSOAR content team. This document identifies the services required and the expected level of services.
+
+#### There are three options for contributing content to the Cortex XSOAR marketplace:
+
+   1. Contribute from the Cortex XSOAR UI - for more information please see [Contributing through the Marketplace](../contributing/marketplace).
+   
+   2. Contribute through a GitHub Pull Request on the public [XSOAR Content Repository](https://github.com/demisto/content) - for more information please see [Contributing docs](../contributing/contributing#how-to-contribute).
+
+   3. Contribute through a private GitHub repository: this is required if you are providing a **Premium** (aka Paid) Content Pack - for more information please see [here](../packs/premium_packs).
+   
+
+In each of the above options, the review phase will begin with the **opening of the GitHub Pull Request** containing your content changes.
+
+#### Once your PR is open, the XSOAR content team commits to the following:
+
+   1. A reviewer will be assigned to your PR within an **hour** after opening it.
+   
+   2. Your reviewer will publish his initial response to your changes within **5 business days**.
+   
+   3. If you were required to make changes to the code, you have made the necessary changes, and have updated about it in a message on the PR - your reviewer will respond within **3 business days**.
+   
+   4. Your reviewer will be available for any questions during the review process - you can contact him on the PR itself or on slack (DFIR community).
+   
+   5. Once your PR is approved and merged by your reviewer, an **internal PR** including your changes will be opened within an **hour**.
+    The internal PR allows us to run our internal validity and security checks on your final code. 
+   
+   6. Once the **internal PR is merged** your changes will be published in the XSOAR marketplace within **3 business days**.
```

Comment:
As we don't commit to any timeline regarding merging the internal PR I don't think we should mention it, instead IMO we should only say that once their PR is merged the changes will be published in 3 business days.

Reply:
They will not as you tell them it will take up to 3 days

Reply:
3 business days after the merge of the internal PR - I'm trying to emphasize here that the tests we run on the internal PR are a step without a time commitment from our side.